src/language.js

```diff
@@ -231,6 +231,15 @@ const protocolTitles = {
 }
 
 const pages = {
+  userDoesntHaveProjectAccess: {
+    title: 'You do not have permission to access this project.',
+    getSubtitle: (projectName) => {
+      const projectNameToUse = projectName || <code>unknown project name</code>
```

Comment:
volunteered some dev UX here for when for whatever reason we cant retrieve an actual project name.